src/PrestaShopBundle/Resources/views/Admin/Improve/Payment/PaymentMethods/payment_methods.html.twig

```diff
@@ -27,6 +27,23 @@
 
 {% block content %}
   <div class="container">
+
+    {% if paymentModules|length < 2 %}
+    <div class="card-block row">
+      <div class="card-text">
+        <div class="row">
+          <div class="col-sm">
+            <div class="alert alert-info" role="alert">
+              <div class="alert-text">
+                {{ 'It is recommended to provide at least two different payment methods. Only one payment method could be problematical if this option cannot be used by a customer and prevent them from ordering.'|trans({}, 'Admin.Payment.Notification') }}
```

Comment:
```suggestion
                {{ 'We recommend providing at least two different payment methods. Only one payment method could be problematic if this option cannot be used by a customer because it will prevent him/her from ordering.'|trans({}, 'Admin.Payment.Notification') }}
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/17818/commits/6526be0c8e0ad45c5b6d3fff8b5a1c7c95bea01e